examples/ads.amp.html

```diff
@@ -785,6 +786,13 @@ <h2 id="smartadserver">SmartAdServer ad</h2>
     <div fallback></div>
   </amp-ad>
 
+  <h2 id="smartclip">smartclip</h2>
+  <amp-ad width=400 height=225
+    type="smartclip"
+    data-plc="84555"
+    data-sz="400x320">
```

Comment:
please follow the example to add placeholder and fallback.
https://github.com/ampproject/amphtml/blob/master/examples/ads.amp.html#L165